cli/targets/static.js

```diff
@@ -19,9 +19,24 @@ var config = {};
 
 static_target.description = "Static code without reflection (non-functional on its own)";
 
+function addImports(root) {
+    if (!root.imports) {
+        return;
+    }
+    root.imports.forEach(function(oneImport) {
+        let name = oneImport;
+        const lastSlashIndex = name.lastIndexOf('/');
+        if (lastSlashIndex != -1) {
+            name = name.substring(lastSlashIndex + 1);
+        }
+        push("const " + name + "_pb = require(\"" + oneImport + "\");\n");
+    });
```

Comment:
1. Please use either single, or double quotes everywhere in strings for consistency
2. Please use template strings instead of using an old syntax with string concatenation 
```suggestion
    imports.forEach((import) => {
        const name = import.replace(/\/$/, '');
        push(`const ${name}_pb = require("${import}");\n`);
    });
```